sidebars.ts

```diff
@@ -1899,6 +1899,7 @@ integrations: [
          'integrations/microsoft-azure/windows-json',
          'integrations/microsoft-azure/windows-legacy',
          'integrations/microsoft-azure/performance',
+         'integrations/microsoft-azure/azure-batch'
```

Comment:
Move the 'integrations/microsoft-azure/azure-batch' line after 'integrations/microsoft-azure/audit' so that the left nav entries are in alphabetical order.